### PR TITLE
feat(): Add function to drop all keys whose values are undefined in a…

### PR DESCRIPTION
**Pull request by huy-nguyen**
Add function to drop all keys whose values are undefined in an object. This is analogous to `removeEmpty` but works for objects.

### CODE REVIEW

File: src/remove-empty-keys.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+export {
```
Comment (kentcdodds) [inline src/remove-empty-keys.js]: Could you add `export * from './remove-empty-keys'` to [`index.js`](https://github.com/kentcdodds/webpack-config-utils/blob/master/src/index.js)?

Comment (kentcdodds) [inline src/remove-empty-keys.js]: Also, what do you think about renaming this to `remove-empty-properties`?

Comment (huy-nguyen) [inline src/remove-empty-keys.js]: Done on both counts.

**Comment by huy-nguyen** [comment]
Yes, `webpack` does. One example I've been able to find: does require all properties in `entry` to be either a string, an array of strings or an object.

Suppose in dev builds, instead of using `removeEmptyProperties` to remove the `undefined` entry `css`:
```javascript
  entry: removeEmptyProperties({
     app: ifProd('./indexWithoutCSS', './indexWithCSS'),
     css: ifProd('./style.scss')
  }),
```
we let the `css` entry be `undefined`:
```javascript
  entry: {
     app: ifProd('./indexWithoutCSS', './indexWithCSS'),
     css: ifProd('./style.scss')
  },
```
`webpack` will complain:
```
Invalid configuration object. Webpack has been initialised using a configuration object that does not match the API schema.
 - configuration.entry['css'] should be one of these:
   string | [string]
   The entry point for one output file.
 - configuration.entry should be one of these:
   object { <key>: string | [string] } | string | [string]
   The entry point(s) of the compilation.
```

**Comment by kentcdodds** [comment]
Ah, right the new schema validation in webpack 2. Thanks!

**Comment by kentcdodds** [comment]
Oh! You know what! It would actually be even better to enhance the existing `removeProperties` to work with either an array or an object. Rather than having to have two functions. I've already merged this, but I stopped the auto-release from happening. Could you make that change? Then we can release that :)

**Pull request merged**